src/Admin/Views/Users/Edit.cshtml

```diff
@@ -45,10 +46,15 @@
                 }
 
                 if (gateway.value === '@((byte)Bit.Core.Enums.GatewayType.Stripe)') {
-                    window.open('https://dashboard.stripe.com/customers/' + customerId.value, '_blank');
+                    const url = '@(HostingEnvironment.IsDevelopment()
+                                     ? "https://dashboard.stripe.com/test"
```

Comment:
We can move the urls to  variables and use the variables within the code so we have a single place to replace the pr for future changes.

Reply:
I pushed a change yesterday which should accomplish this! Let me know what you think